front/types/gens.ts

```diff
@@ -12,3 +12,8 @@ export type GensRetrievedDocumentType = {
     score: number;
   }[];
 };
+
+export type GensTemplateType = {
+  name: string;
+  instructions: string[]
```

Comment:
Let's add `color` here

Reply:
Add `sId` as well and we probably want to add `userId` and `workspaceId` for convenience (numbers)

Ideally we implement all the DB operations in `lib/api/gens.ts` (see chat for an example)